src/model_operations.py

```diff
@@ -52,21 +72,24 @@ async def generate_tokens(prompt, model):
         print(reason)
         yield reason
         return
-    
+
     try:
-        tokens = model.tokenize(prompt.encode())  # 'prompt.encode()' converts the string to bytes."
-    except Exception as e:
+        tokens = model.tokenize(
+            prompt.encode()
+        )  # 'prompt.encode()' converts the string to bytes."
```

Comment:
Probably a personal perception, but I don't find find this type of formatting better.

Reply:
I understand your point. The formatting was automatically applied by Black, which we’re using to maintain consistency across the codebase. However, if you believe there’s a better way to format this, I’m open to discussing it and adjusting the code for better readability or to better match our preferences.

Reply:
Yes, that's what I thought. This is one of the reasons why I was avoiding automation over the code syntax. I don't find any particular value on this. I'm fine if you want to keep it.